cves/CVE-2017-7675.yml

```diff
@@ -173,8 +203,10 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
```

Comment:
Consider revising to "...input not being properly parsed..." for smoother reading.

Reply:
Will do!